icc-x-api/storage/LocalStorageFacade.ts

```diff
@@ -0,0 +1,52 @@
+export interface LocalStorageFacade {
```

Comment:
That's exactly what I meant : Do not link this facade too much to the LocalStorage. In fact, seeing the usage of it, I would even do two facades : One dedicated to the keys management and one dedicated to other information (like the date we're saving). 
I would update it and do something more generic, where LocalStorageImpl could be a proper implementation. 
For key management, I would do something like : 
- `getPublicKey(keyId: string): string?`
- `getPrivateKey(keyId: string): string?`
- `getKeypair(keyId: string): { privKey: string, pubKey: string } | undefined`
- `storeKeypair(keyId: string, keypairToStore: { privKey: string, pubKey: string }): void`
- `cleanKeypair(keyId: string): void`

For Storage facade, something like you did seems ok, but I would use generics: 
- `getItem<T>(keyId: string): T?`
- `setItem(itemId: string, itemToStore: T): void`
- `cleanItem(itemId: string): void`